CHANGELOG.md

```diff
@@ -18,9 +18,11 @@ This can also be enabled programmatically with `warnings.simplefilter('default',
 
 ## [2.5.8] - not released yet
 ### Added
+- support for description list (`<dl>`), description titles (`<dt>`), and description details (`<dd>`).
 - support for monochromatic images (PIL `image.mode == '1'`) thanks to @GerardoAllende
 - demonstration Jupyter notebook: [tutorial/notebook.ipynb](https://github.com/PyFPDF/fpdf2/blob/master/tutorial/notebook.ipynb)
 ### Fixed
+- `HTMLMixin` is deprecated. `write_html` method is now available in `FPDF` class
```

Comment:
```suggestion
- `HTMLMixin` is deprecated. `write_html` method is now available in `FPDF` class - thanks to @yk-jp 
```

Reply:
apply the suggestion at https://github.com/PyFPDF/fpdf2/commit/0dfbfaeb18a549c3e73cfacc51bc50708f2322c6